Real-Dev-Squad/website-my
# Chore/fix merge conflicts

### Developer Name: Joy Gupta

### Purpose
Resolve merge conflicts between develop and main

### Steps I Followed
1. Synced `develop and main` branch of clone repo with RDS
2. Took a pull of origin (cloned repo)
3. Created a branch from `develop` named `chore/fix-merge-conflicts`
4. Rebased it with the `upstream's main` 

### PR Number(s): 511

### Frontend Changes

### Is Under Feature Flag

### Database changes: 

### Breaking changes (If your feature is breaking/missing something please mention pending tickets): 

### Deployment notes:
NA

### Tested in local

### QA Instructions, Screenshots, Recordings

Run `yarn test` and there you can see test case running.
![image](https://github.com/Real-Dev-Squad/website-my/assets/56365512/749f4744-e6cb-4177-97e7-802eacb53c6b)
